conf/application.conf

```diff
@@ -147,12 +147,18 @@ external-url {
 services-config = [
     {
         name: "atar",
-        enrolment: "HMRC-ATAR-ORG"
+        enrolment: "HMRC-ATAR-ORG",
+        callBack: "/advance-tariff-application",
+        friendlyName: "Advance Tariff Rulings",
```

Comment:
We will need `shortName:  "ATaR"` in there as well.

Reply:
Yeah, I'll add this too